services/streammanager/lambda/lambdamanager_test.go

```diff
@@ -81,26 +82,6 @@ func TestProduceWithInvalidData(t *testing.T) {
 	assert.Equal(t, 400, statusCode)
 	assert.Equal(t, "Failure", statusMsg)
 	assert.Contains(t, respMsg, "[Lambda] error :: Invalid payload")
-
-	// Destination Config not present
-	sampleEventJson, _ = json.Marshal(map[string]interface{}{
-		"payload": sampleMessage,
-	})
-	statusCode, statusMsg, respMsg = producer.Produce(sampleEventJson, map[string]string{})
-	assert.Equal(t, 400, statusCode)
-	assert.Equal(t, "Failure", statusMsg)
-	assert.Contains(t, respMsg, "[Lambda] error :: Invalid destination config")
-
-	// Invalid Destination Config
-	sampleDestConfig := map[string]interface{}{}
-	sampleEventJson, _ = json.Marshal(map[string]interface{}{
-		"payload":    sampleMessage,
-		"destConfig": "invalid dest config",
-	})
-	statusCode, statusMsg, respMsg = producer.Produce(sampleEventJson, sampleDestConfig)
-	assert.Equal(t, 400, statusCode)
-	assert.Equal(t, "Failure", statusMsg)
-	assert.Contains(t, respMsg, "[Lambda] error while unmarshalling jsonData")
```

Comment:
Why are we removing these tests?

Reply:
Destination config is not part the input message any more so this is not required anymore